node/config.go

```diff
@@ -360,7 +360,8 @@ func (c *Config) parsePersistentNodes(path string) []*discover.Node {
 	return nodes
 }
 
-func makeAccountManager(conf *Config) (*accounts.Manager, string, error) {
+// ResolveAccountConfig determines the settings for scrypt and keydirectory
+func ResolveAccountConfig(conf *Config) (int, int, string, error) {
```

Comment:
Why not just make `ResolveAccountConfig` a method on `Config`? Also I'd rename to simply `AccountConfig`.